hoomd/Updater.h

```diff
@@ -118,7 +118,6 @@ namespace detail
 void export_Updater(pybind11::module& m);
 
     } // end namespace detail
-
```

Comment:
Please revert this change. We don't want to commit changes that aren't relevant to the PR to make it easier to read.